response_operations_ui/static/minimised/all.css.min

```diff
@@ -0,0 +1 @@
+#test-div{font-size:5rem}
```

Comment:
These `.min` files don't need to be checked in
I think the minimised folder is in `.gitignore`?

Reply:
Wrong file name hence why it was checked in. I've added the folder instead